tests/unit/test_gff3_parser.py

```diff
@@ -0,0 +1,102 @@
+"""
+Data-driven tests for rna quantification.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import ga4gh.gff3 as gff3
+import unittest
+
+
+class TestGff3ParserOnTypicalFile(unittest.TestCase):
+    """
+    Data driven unit tests for the GFF3 parser
+    """
+    def setUp(self):
```

Comment:
OK, let's forget about this then. Reasonable test coverage over the subset of GFF we're actually using will do fine. These data-driven test cases can condition on known properties of the specific data files; this should be a quick way of getting the parser done with decent test coverage.